staging/src/k8s.io/apimachinery/pkg/util/validation/field/errors.go

```diff
@@ -33,6 +33,13 @@ type Error struct {
 	Field    string
 	BadValue interface{}
 	Detail   string
+
+	// Origin indicates the source of this error. It is used in testing to
+	// ensure that the expected errors are returned without matching on detail
+	// strings.
```

Comment:
```suggestion
	// Origin uniquely identifies the source of an error. It is used in testing to
	// to compare expected errors against actual errors independent on contents
	// of detail strings.
```

?

Reply:
"source" is probably the wrong word.  "cause" ?  "reason for"?  "trigger" ?

also "independent on contents of detail strings" is awkward?

Reply:
:+1: New working LGTM